### ISSUE TITLE
Upgrade to lodash@^4.0.0

### ISSUE THREAD

**Issue by cbrunnkvist** [city41/node-sql-fixtures#38]
Npm complains

```
npm WARN deprecated lodash@2.4.2: lodash@<3.0.0 is no longer maintained. Upgrade to lodash@^4.0.0
```

**Comment by city41** [comment]
I've started doing this, but really all of sql-fixtures packages need an upgrade. I'm going to try and get this done by this weekend.

**Comment by city41** [comment]
Resolved with 0.13.0. upgraded to lodash 4.3.0, knex 0.9.0 and bluebird 3.2.2
